CHANGELOG.md

```diff
@@ -10,6 +10,9 @@
   - `ActionMenuPrimaryButton`
   - `ActionMenuSecondaryButton`
   - `ActionMenuFlatButton`
+  - `Spinner` no longer has property `large` and no related class exist.
```

Comment:
9.0.0 has already been released.

Create this at the top:

```
## Upcoming
```

Reply:
> 9.0.0 has already been released.
> 
> Create this at the top:
> 
> ```
> ## Upcoming
> ```

Fixed!